### PR TITLE
[202012] Replace swsssdk with swsscommon in sonic-host-services and update submodule sonic-swss-common

### PR DESCRIPTION
**Pull request by qiluo-msft**
#### Why I did it
Backport https://github.com/Azure/sonic-buildimage/pull/8034 to 202012 branch

sonic-swss-common submodule updating includes below commits
```
a6b98da 2021-04-29 | Add support for config_db subscribe and unsubscribe python apis (#481) [arlakshm]
2506ca0 2021-08-22 | [ci] Fix azure pipeline DownloadPipelineArtifact source branch (#514) [Qi Luo]
```

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

File: src/sonic-host-services/tests/hostcfgd/hostcfgd_test.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import os
 import sys
-import swsssdk
+import swsscommon
```
Comment (sujinmkang) [inline src/sonic-host-services/tests/hostcfgd/hostcfgd_test.py:3]: just minor cosmetic changes, same as the changes in other files, can you change this to "from swsscomm import swsscommon"?

File: src/sonic-host-services/tests/hostcfgd/hostcfgd_test.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 from .mock_configdb import MockConfigDb
 
 
-swsssdk.ConfigDBConnector = MockConfigDb
+swsscommon.swsscommon.ConfigDBConnector = MockConfigDb
```
Comment (sujinmkang) [inline src/sonic-host-services/tests/hostcfgd/hostcfgd_test.py:13]: And also change this line to just "swsscommon.ConfigDBConnector = MockConfigDb"?

File: src/sonic-host-services/tests/hostcfgd/hostcfgd_test.py (hunk 1 above)
Comment (qiluo-msft) [reply src/sonic-host-services/tests/hostcfgd/hostcfgd_test.py:3]: I submit another PR on master to solve your comments: [https://github.com/Azure/sonic-buildimage/pull/8515](https://github.com/Azure/sonic-buildimage/pull/8515).

Keep this PR as simple as backport.

**Review by sujinmkang — APPROVED** [review]

**Pull request merged**